specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/LUIS-Authoring.json

```diff
@@ -7862,6 +7890,256 @@
+        ],
+        "x-ms-examples": {
+          "Successful Import Application": {
+            "$ref": "./examples/versions/SuccessfulImportV2ApplicationVersionRequest.json"
+          }
+        }
+      }
+    },
+    "/apps/{appId}/versions/import?format=lu": {
+      "post": {
+        "description": "Imports a new version into a LUIS application.",
+        "operationId": "Versions_ImportLuFormat",
+        "parameters": [
+          {
+            "$ref": "#/parameters/AppIdInPath"
+          },
+          {
+            "name": "versionId",
+            "in": "query",
+            "description": "The new versionId to import. If not specified, the versionId will be read from the imported object.",
+            "type": "string"
+          },
+          {
+            "name": "luisAppLu",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "An LU representing the LUIS application structure."
+          }
+        ],
+        "consumes": [
+          "text/plain"
+        ],
+        "responses": {
+          "201": {
+            "description": "The created application version.",
+            "schema": {
+              "$ref": "#/definitions/VersionResponse"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Successful Import Application": {
+            "$ref": "./examples/versions/SuccessfulImportLuApplicationVersionRequest.json"
+          }
+        }
+      }
+    },
+    "/apps/{appId}/versions/{versionId}/export?format=lu": {
+      "get": {
+        "description": "Exports a LUIS application to text format.",
+        "operationId": "Versions_ExportLuFormat",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "format",
+            "required": true,
+            "description": "Lu format extension",
+            "type": "string",
+            "enum": [
+              "lu"
+            ]
+          },
+          {
+            "$ref": "#/parameters/AppIdInPath"
+          },
+          {
+            "$ref": "#/parameters/VersionIdInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The LUIS application structure in LU format.",
+            "schema": {
+              "description": "The LUIS application structure in LU format.",
+              "type": "object",
+              "format": "file"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "produces": [
+          "application/octet-stream"
```

Comment:
Just to confirm it's not text/plain (since I see at creation time it's text/plain)

Reply:
yes, it  "application/octet-stream"